Dockerfile

```diff
@@ -403,10 +407,23 @@ ENV SIGNALFX_BUNDLE_DIR=/bundle \
     LC_ALL=C.UTF-8 \
     LANG=C.UTF-8
 
+RUN curl -fsSL get.docker.com -o /tmp/get-docker.sh &&\
+    sh /tmp/get-docker.sh
+
+# Get integration test deps in here
 RUN pip3 install ipython ipdb
+COPY python/setup.py /tmp/
+RUN pip3 install -e /tmp/
```

Comment:
I am skeptical of this.  In general the developer should be either using the python runner in the bundle dir in the dev image or referencing the packages directly in the `python` directory from the shared mount of the agent code (that is what the `PYTHONPATH=/usr/src/signalfx-agent/python` above accomplishes).  What were you trying to solve here?

Reply:
Ok, I see it is just copied from below.  Still, it doesn't seem right.

Reply:
> Ok, I see it is just copied from below. Still, it doesn't seem right.

Yeah, not sure why it was done this way to begin with.  Will change to install using python from the bundle dir instead.

Reply:
These lines have been removed.